ui-v2/app/styles/base/reset/system.scss

```diff
@@ -21,6 +21,9 @@ html {
 hr {
   background-color: $gray-500;
 }
+button {
+  background-color: $transparent;
+}
```

Comment:
This was only being applied on buttons in the main nav so that the native gray wouldn't hide the background brand color of the wrapping label. Really, we don't want buttons to have native gray anywhere in the app so I moved it to our bulma inherited resets.